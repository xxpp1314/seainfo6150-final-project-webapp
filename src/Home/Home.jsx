import React from 'react'
import styles from "./Home.module.css";
import PetCategories from "../PetCategories/PetCategories.jsx"
import RecommendPets from "../RecommendPets/RecommendPets.jsx";

const Home = () => {
    return (
        <div className={styles.homepage}>
            <PetCategories/>
            <RecommendPets/>    
        </div>
    )
}

export default Home
