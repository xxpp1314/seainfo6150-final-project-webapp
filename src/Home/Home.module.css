.homepage{ 
    background-image: linear-gradient(to right,rgb(83, 15, 15), rgb(49, 12, 25) 70%);
}