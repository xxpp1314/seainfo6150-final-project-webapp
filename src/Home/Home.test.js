import React from "react";
import { render } from "@testing-library/react";
import Home from "./Home.jsx";
import { BrowserRouter } from "react-router-dom";

it("renders the Home Page correctly", () => {
  const { container } = render(
    <BrowserRouter>
      <Home/>
    </BrowserRouter>
  );
  expect(container).toMatchSnapshot();
});
