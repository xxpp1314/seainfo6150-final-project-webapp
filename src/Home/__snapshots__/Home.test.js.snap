// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the Home Page correctly 1`] = `
<div>
  <div
    class="homepage"
  >
    <div
      class="container"
    >
      <div
        class="flex"
      >
        <img
          alt=""
          class="image"
          src="dogs.jpg"
        />
        <p
          class="text"
        >
          <a
            class="text"
            href="/dogs"
          >
            Dogs
          </a>
        </p>
      </div>
      <div
        class="flex"
      >
        <img
          alt=""
          class="image"
          src="cats.jpg"
        />
        <p
          class="text"
        >
          <a
            class="text"
            href="/cats"
          >
            Cats
          </a>
        </p>
      </div>
      <div
        class="flex"
      >
        <img
          alt=""
          class="image"
          src="rabbits.jpg"
        />
        <p
          class="text"
        >
          <a
            class="text"
            href="/rabbits"
          >
            Rabbits
          </a>
        </p>
      </div>
      <div
        class="flex"
      >
        <img
          alt=""
          class="image"
          src="birds.jpg"
        />
        <p
          class="text"
        >
          <a
            class="text"
            href="/birds"
          >
            Birds
          </a>
        </p>
      </div>
      <div
        class="flex"
      >
        <img
          alt=""
          class="image"
          src="others.jpg"
        />
        <p
          class="text"
        >
          <a
            class="text"
            href="/others"
          >
            Other Pets
          </a>
        </p>
      </div>
    </div>
    <div>
      <h1
        class="title"
      >
        Find Your Suitable Pet Here
      </h1>
      <div
        class="container"
      >
        <div
          class="grid"
        >
          <div
            class="div"
          >
            <img
              alt=""
              class="image"
              src="Bane.jpg"
            />
            <a
              class="text"
              href="/dogs/Bane"
            >
              Smart Dog: Bane
            </a>
          </div>
          <div
            class="div"
          >
            <img
              alt=""
              class="image"
              src="Milo.jpg"
            />
            <a
              class="text"
              href="/cats/Milo"
            >
              Cute Cat: Milo
            </a>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
